### DISCUSSION TITLE
Custom layout with figure spanning multiple rows

### DISCUSSION THREAD

**Discussion by tomshafer** [quarto-dev/quarto-cli#1233] [Q&A] [answered]
Hi, 

I'm really enjoying Quarto! I can't figure out, though, how to create a three-figure layout where two figures are stacked on the left and the third spans the right. The documentation demonstrates the horizontal case with `#| layout: [[1, 1], [1]]`, but it isn't clear how to do the transposition.

Many thanks for the help.

**Comment by jjallaire** [comment, marked as answer]
There unfortunately isn't a way to do this right now.

**Comment by tomshafer** [comment]
That explains it, then. Many thanks!

**Reply by YinCY123** [reply]
Is there are any solutions right now? 2024/11

**Reply by mcanouil** [reply]
No, there isn't.
`layout` is a row-wise system.

---

edit: you can use whatever underlying technology the format you are targeting is using, for example, if you target HTML (not Reveal.js):

````qmd
---
title: "Quarto Playground"
format: html
---

::: {.grid}

::: {.g-col-6}
![]({{< placeholder 600 800 >}})
:::

::: {.g-col-6}
::: {.grid}
::: {.g-col-12}
![]({{< placeholder 600 400 >}})
:::
::: {.g-col-12}
![]({{< placeholder 600 400 >}})
:::
:::
:::

:::
````

See https://quarto.org/docs/output-formats/page-layout.html#css-grid
